swift-ci/master/debian/9/Dockerfile

```diff
@@ -30,7 +30,8 @@ RUN apt -y install      \
   tzdata                \
   unzip                 \
   vim                   \
-  uuid-dev
+  uuid-dev		\
+  ca-certificates
```

Comment:
fwiw, the Debian 9 dockerfile is still one we are working on, hence we did not publish toolchain for debian yet.